purcahseSection.js

```diff
@@ -0,0 +1,296 @@
+    </g>
+  </svg></span>`;
+      
+      subContent.appendChild(detailsConatiner);
+ 
+      
+      listItem.innerHTML=`<span class="discount-info">$5 off every order</span>`;
+  list.prepend(listItem);
+  }
+    function setOtpContainer(i)
+      {
+        const svgIconNew = document.createElement("div");
+    svgIconNew.classList.add("radio-select-new");
+    
+    svgIconNew.innerHTML = `
+      <svg xmlns="http://www.w3.org/2000/svg" id="calendar-icon" width="21.027" height="21.078" viewBox="0 0 21.027 21.078">
+        <path id="Path_70167" data-name="Path 70167" d="M0,0H17.434V17.434H0Z" fill="none"/>
+        <g id="Ellipse_484" data-name="Ellipse 484" transform="translate(0.027 0.078)" fill="#fff" stroke="#c70072" stroke-width="1.5">
+          <circle cx="10.5" cy="10.5" r="10.5" stroke="none"/>
+          <circle cx="10.5" cy="10.5" r="9.75" fill="none"/>
+        </g>
+        <g id="Ellipse_485" data-name="Ellipse 485" transform="translate(4.027 4.078)" fill="#c70072" stroke="#c70072" stroke-width="1">
+          <circle cx="6.5" cy="6.5" r="6.5" stroke="none"/>
+          <circle cx="6.5" cy="6.5" r="6" fill="none"/>
+        </g>
+      </svg>
+    `;
+                const perCapsulepriceWrrapper=document.createElement("div");
+          perCapsulepriceWrrapper.classList.add("price-container-new");
+      const price=skioContainers[i].querySelector(".price-span");
+         const priceContent=skioContainers[i].querySelector(".price-span");
+          const priceTextOtp = priceContent.textContent.trim();
+    const priceRegex = /\$([\d.]+)/;
+    const match = priceTextOtp.match(priceRegex);
+    const priceValue = match ? match[1] : "";
+         price.innerHTML=`$${priceValue}+ $7.99 <span class="shipping-text"> (shipping)</span>`;
+      const pricePerCapsule=document.querySelector(".per-capsule-otp-price span");
+     
+        const pricePerCapsuleSpan=document.createElement("span");
+        pricePerCapsuleSpan.classList.add("price-per-capsule");
+        pricePerCapsuleSpan.textContent=pricePerCapsule.textContent;
+            const PricePerCapsuleSpanParentDiv=document.createElement("div");
+         PricePerCapsuleSpanParentDiv.appendChild(pricePerCapsuleSpan);
+         PricePerCapsuleSpanParentDiv.classList.add("per-capsule-otp-price");
+      perCapsulepriceWrrapper.appendChild(price);
+      perCapsulepriceWrrapper.appendChild(PricePerCapsuleSpanParentDiv);
+         
+           const skioTitle=skioContainers[i].querySelector(".skio-onetime-group-title");
+         
+      skioTitle.appendChild(perCapsulepriceWrrapper);
+         
+         skioTitle.prepend(svgIconNew);
+      }
+    
+  for(let i=0;i<skioContainers.length;i++)
+  {
+     
+  
+  if(VarientsContainer)
+  {
+  
+    //const parentElement = skioContainers[i].querySelector(".skio-group-title");
+    //parentElement.prepend(svgIconNew);
+    if(i==0 || i==1)
```

Comment:
instead of checking if with i=0 or 1 apply this function to all the elemets expect the last